src/helpers/productStock.js

```diff
@@ -0,0 +1,11 @@
+import {inventory} from "../constants/inventory.js";
```

Comment:
Je wilt nooit de array hier zelf importeren. Deze 'helper' functie werkt nu namelijk alleen maar voor deze array en is dus niet herbruikbaar voor andere arrays.

Om een helper functie herbruikbaar te maken wil je gebruik maken van paramaters. Wanneer je vervolgens de helper functie aanroept in een ander component (in dit geval app.jsx) kan je de gewenste array als argument meegeven,

In je helper:

function productStock(array) {
    const totalStock = array.reduce((acc, tv) => {
        return acc + tv.originalStock;
    }, 0);

    return totalStock;
}

export default productStock;

In app.jsx:

productStock(inventory)